modules/Core/src/main/java/org/terasology/logic/inventory/ItemCommands.java

```diff
@@ -49,27 +49,37 @@
     @In
     private AssetManager assetManager;
 
-    @Command(shortDescription = "Adds an item to your inventory", runOnServer = true,
+
+    @Command(shortDescription = "Adds an item or block to your inventory",
+            helpText = "Puts the desired number of the given item or block with the given shape into your inventory",
+            runOnServer = true,
             requiredPermission = PermissionManager.CHEAT_PERMISSION)
-    public String giveItem(
+    public String give(
             @Sender EntityRef client,
             @CommandParam("prefabId or blockName") String itemPrefabName,
-            @CommandParam(value = "amount", required = false) Integer amount) {
+            @CommandParam(value = "amount", required = false) Integer amount,
+            @CommandParam(value = "blockShapeName", required = false) String shapeUriParam) {
+
+        int itemAmount = amount != null ? amount : 1;
         Set<ResourceUrn> matches = assetManager.resolve(itemPrefabName, Prefab.class);
         switch(matches.size()) {
             case 0:
-                return "Could not find any item matching \"" + itemPrefabName + "\"";
+                return blockCommands.giveBlock(client, itemPrefabName, amount, shapeUriParam);
```

Comment:
I'm still thinking about a way to handle this delegation "better". Another might be to indicate that no item was found right here, prepending the info to the output of `giveBlock`:
```Java
return "Could not find a matching item for \"" + itemPrefabName + "\". \n" + blockCommands.giveBlock(client, itemPrefabName, amount, shapeUriParam);
```